### PR TITLE
update default vault spread to 30bps

### PR DESCRIPTION
**Pull request by tqin7**
### Changelist
updated spread to 30bps, latest recommended value

### Test Plan
N/A

### Author/Reviewer Checklist
- [ ] If this PR has changes that result in a different app state given the same prior state and transaction list, manually add the `state-breaking` label.
- [ ] If the PR has breaking postgres changes to the indexer add the `indexer-postgres-breaking` label.
- [ ] If this PR isn't state-breaking but has changes that modify behavior in `PrepareProposal` or `ProcessProposal`, manually add the label `proposal-breaking`.
- [ ] If this PR is one of many that implement a specific feature, manually label them all `feature:[feature-name]`.
- [ ] If you wish to for mergify-bot to automatically create a PR to backport your change to a release branch, manually add the label `backport/[branch-name]`.
- [ ] Manually add any of the following labels: `refactor`, `chore`, `bug`.

## Summary by CodeRabbit

- **Bug Fixes**
  - Adjusted the `spread_min_ppm` parameter in the vault module from 10000 to 3000 to improve spread calculations, potentially enhancing pricing and trading strategies.

### CODE REVIEW

File: protocol/x/vault/types/params.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ import (
 func DefaultParams() Params {
 	return Params{
 		Layers:                           2,                            // 2 layers
-		SpreadMinPpm:                     10_000,                       // 100 bps
+		SpreadMinPpm:                     3_000,                        // 30 bps
```
Comment (vincentwschau) [inline protocol/x/vault/types/params.go:13]: nit: Is there a need to change this at all?

Comment (tqin7) [reply protocol/x/vault/types/params.go:13]: wanted dev/staging to mirror spread in prod

Comment (vincentwschau) [reply protocol/x/vault/types/params.go:13]: Can you do that via the genesis rather than via the code here?

Comment (tqin7) [reply protocol/x/vault/types/params.go:13]: sure that works too

**Pull request closed without merging**